README.md

```diff
@@ -229,23 +229,55 @@ local kp =
     },
   };
 
-{ 'setup/0namespace-namespace': kp.kubePrometheus.namespace } +
```

Comment:
There is no need to change the README to add those functions. I think README needs to include only the very basics for someone to get started with kube-prometheus, and modifying alerts is not something that everyone will need on their first day.

Also, this snippet is auto-generated when running `make generate`, that's why CI is failing 🙂

Reply:
Thanks for the review :) I will restore this section and move these descriptions to "Changing Default Rules" section.